frontend/package.json

```diff
@@ -9,7 +9,7 @@
 		"lint": "eslint --ext .js,.vue ./",
 		"dev": "quasar dev",
 		"build": "quasar build",
-		"sonar": "sonar-scanner",
+		"sonar": "node_modules/sonar-scanner/bin/sonar-scanner",
```

Comment:
Not a huge fan to have two way to call sonar in a project. I think is better to remove this.